Clinical-Genomics/cg
# Gets dates from status db instead of LIMS

This PR has a delivery report that relies on status db for dates instead of LIMS

How to test:
0. install on stage (add instructions on how to as needed)
1. us
1. find/create a family that does exist in status-db but is missing received, prepared, sequenced and delivered dates in LIMS
1. cg upload delivery-report -p [crazyowl]

Expected outcome:
dates for received, prepared, sequenced and delivered exist in the report
